src/vma/sock/sock-redirect.cpp

```diff
@@ -2107,6 +2107,12 @@ pid_t fork(void)
 
 		safe_mce_sys().get_env_params();
 		vlog_start("VMA", safe_mce_sys().log_level, safe_mce_sys().log_filename, safe_mce_sys().log_details, safe_mce_sys().log_colors);
```

Comment:
@rafiw Please explain why rdma_lib_reset() should not be called in daemon() API as well